### PR TITLE
[CMR] M3-4206 pt3: Prevent text from being flush to the screen

### PR DESCRIPTION
**Pull request by tiffwong**
## Description
This PR will address the feedback from [https://github.com/linode/manager/pull/6853](https://github.com/linode/manager/pull/6853) which includes:
- [x] Linode Details: Backups / Activity Logs / Settings tabs
- [x] StackScripts Landing
- [x] Lonview Details
- [x] Profile
- [x] Search Landing
- [x] Support Ticket Landing

**Updated:**
- [x] Linode Details: Activity Log tab
- [x] Longview Landing: Clients tab

## Type of Change
- Non breaking change ('update', 'change')

### CODE REVIEW

**Review by DevDW — COMMENTED** [review]
Nice work; I noticed a couple of small things:

1. The filter box and "Sort by" label are flush to the screen on the Longview landing page
Screen Shot 2020-09-16 at 12 49 10 PM

2. "Duration" looks like it has an extra space in front of it
Screen Shot 2020-09-16 at 12 47 30 PM

File: packages/manager/src/features/Search/SearchLanding.tsx [hunk 1]
```diff
@@ -196,7 +203,7 @@ export const SearchLanding: React.FC<CombinedProps> = props => {
               {query && splitWord(query)}
             </Typography>
             <Typography style={{ marginTop: 56 }} className="nothing">
-              sorry, no results for this one
+              Sorry, no results for this one
```
Comment (johnwcallahan) [inline packages/manager/src/features/Search/SearchLanding.tsx:206]: Ha, good catch.

File: packages/manager/src/features/linodes/LinodesDetail/LinodeActivity/LinodeActivity_CMR.tsx [hunk 2]
```diff
@@ -33,7 +36,7 @@ export const LinodeActivity: React.FC<CombinedProps> = props => {
         className={classes.title}
         data-qa-settings-header
       >
-        Activity Feed
+        Activity Logs
```
Comment (johnwcallahan) [inline packages/manager/src/features/linodes/LinodesDetail/LinodeActivity/LinodeActivity_CMR.tsx:39]: I think I missed it... where is this change coming from? Seems like "Activity Log" might be better.

**Review by johnwcallahan — APPROVED** [review]

**Review by WilkinsKa1 — CHANGES_REQUESTED** [review]
Great work on this, I found a few more 😅   but then I think we should be all good:

On Linode Backups tab (at the very bottom):
Screen Shot 2020-09-18 at 3 27 33 PM

On Longview/overview tab:
Screen Shot 2020-09-18 at 3 27 57 PM

Longview/Apache tab (also on MySQL and Nginx):
Screen Shot 2020-09-18 at 3 28 22 PM

File: packages/manager/src/features/linodes/LinodesDetail/LinodeActivity/LinodeActivity_CMR.tsx (hunk 2 above)
Comment (tiffwong) [reply packages/manager/src/features/linodes/LinodesDetail/LinodeActivity/LinodeActivity_CMR.tsx:39]: The tab says Activity Logs so I was trying to make them match 😅

Comment (tiffwong) [reply packages/manager/src/features/linodes/LinodesDetail/LinodeActivity/LinodeActivity_CMR.tsx:39]: I can change the tab to say Activity Feed instead so they'll be consistent

**Review by WilkinsKa1 — APPROVED** [review]
Good to go, thanks for updating!

File: packages/manager/src/features/linodes/LinodesDetail/LinodeSettings/LinodeConfigDrawer.tsx [hunk 3]
```diff
@@ -539,7 +539,6 @@ class LinodeConfigDrawer extends React.Component<CombinedProps, State> {
                 onChange={this.handleRootDeviceChangeTextfield}
                 inputProps={{ name: 'root_device', id: 'root_device' }}
                 fullWidth
-                autoFocus={true}
```
Comment (tiffwong) [inline packages/manager/src/features/linodes/LinodesDetail/LinodeSettings/LinodeConfigDrawer.tsx:542]: Linter recommended removing this

**Pull request merged**